pages/common/dtrx.md

```diff
@@ -0,0 +1,16 @@
+# dtrx
+
+> "Do The Right eXtraction" - extract any archive into new directory by guessing the tool from extension.
+> More information: <https://github.com/dtrx-py/dtrx>.
+
+- Extract archive, guessing the extraction tool from extension:
+
+`dtrx {{foo}}`
+
+- Extract archive, overwrite any existing target output:
+
+`dtrx --overwrite {{foo}}`
+
+- Extract archive, extract everything into current directory:
```

Comment:
```suggestion
- Extract archive and put everything in in into current directory:
```

Reply:
double `in in`

Reply:
Fixed the typo. 
I don't remember if there's anything about this in the style guide, but my suggestion is more in line with the style we use in the rest of the pages. I don't think I've seen this kind of "Title, subtitle" being used anywhere else.